packages/mux-uploader/src/layouts/block.ts

```diff
@@ -3,25 +3,26 @@ import { document } from '../polyfills';
 import '../mux-uploader-file-select';
 import { fileSelectFragment } from '../mux-uploader-file-select';
 
-const template = document.createElement('template');
+export default function blockLayout(noDrop: boolean) {
+  const wrapper = noDrop ? 'div' : 'mux-uploader-drop overlay';
 
-template.innerHTML = /*html*/ `
-<style>
-</style>
+  return document.createRange().createContextualFragment(`
+    <style>
+      /* Add your styles here */
+    </style>
 
-<mux-uploader-drop overlay>
-  <mux-uploader-status></mux-uploader-status>
-  <mux-uploader-retry></mux-uploader-retry>
+    <${wrapper}>
+      <mux-uploader-status></mux-uploader-status>
+      <mux-uploader-retry></mux-uploader-retry>
 
-  <mux-uploader-file-select>
-    <slot name="file-select">
-      ${fileSelectFragment}
-    </slot>
-  </mux-uploader-file-select>
+      <mux-uploader-file-select>
+        <slot name="file-select">
+          ${fileSelectFragment}
```

Comment:
for now this is okay but I find it counter intuitive we need to repeat almost all HTML contents of mux-uploader-file-select here. I will help look at this later.